src/Pact/Analyze/Check.hs

```diff
@@ -248,14 +249,25 @@ describeSmtFailure = \case
     ]
   UnexpectedFailure smtE -> T.pack $ show smtE
 
+describeUnknownFailure :: SBV.SMTReasonUnknown -> Text
+describeUnknownFailure reason = "You've written a hell of a property here. Usually properties are simple things, like \"is positive\" or \"conserves mass\". But not this bad boy. This here property broke the SMT solver. Wish we could help but you're on your own with this one (actually, please report this as an issue: https://github.com/kadena-io/pact/issues).\n\nGood luck...\n" <> tShow reason
+
 describeQueryFailure :: SmtFailure -> Text
 describeQueryFailure = \case
   Invalid model  -> "Wow. We (the compiler) have bad news for you. You know that property / invariant you wrote? It's great. Really. It's just that it divides by zero or somesuch and we don't know what to do with this. Good news is we have a model which may (fingers crossed) help debug the problem:\n" <> showModel model
-  Unknown reason -> "You've written a hell of a property here. Usually properties are simple things, like \"is positive\" or \"conserves mass\". But not this bad boy. This here property broke the SMT solver. Wish we could help but you're on your own with this one (actually, please report this as an issue: https://github.com/kadena-io/pact/issues).\n\nGood luck...\n" <> tShow reason
+  Unknown reason -> describeUnknownFailure reason
   err@SortMismatch{} -> "(QueryFailure): " <> describeSmtFailure err
   Unsatisfiable  -> "Unsatisfiable query failure: please report this as a bug"
   UnexpectedFailure smtE -> T.pack $ show smtE
 
+describeVacuousProperty :: SmtFailure -> Text
+describeVacuousProperty = \case
+  Invalid _ -> "Unexpected solver response during vacuous property check: please report this as a bug"
+  Unknown reason -> describeUnknownFailure reason
+  err@SortMismatch{} -> "Vacuous property check: " <> describeSmtFailure err
+  Unsatisfiable -> "Vacuous property encountered! There is no way for a transaction to succeed if it is calls this function from the top-level. Because all `property` expressions in Pact assume transaction success, in this case it would be possible to validate *any* `property`, even e.g. `false`."
```

Comment:
"if it is calls this function"

Reply:
Good catch -- thanks! Fixing this shortly.